docker-compose.yml

```diff
@@ -1,40 +1,34 @@
 version: "3.8"
 
 services:
-  dev:
-    container_name: nestjs_api_dev
-    image: nestjs-api-dev:1.0.0
+  api:
+    # container_name: nestjs_api_dev
+    # image: nestjs-api-dev:1.0.0
     build:
       context: .
       target: development
-      dockerfile: ./Dockerfile
+      dockerfile: Dockerfile
     command: npm run start:debug
+    depends_on:
+      - postgres
+    environment:
+      DATABASE_URL: postgres://user:pass@postgres:5432/db
+      NODE_ENV: development
+      PORT: 3000
     ports:
-      - 3000:3000
-      - 9229:9229
-    networks:
-      - nesjs-network
-    volumes:
-      - .:/usr/src/app
-      - ./node_modules:/usr/src/app/node_modules
-    restart: unless-stopped
-  prod:
-    container_name: nestjs_api_prod
-    image: nestjs-api-prod:1.0.0
-    build:
-      context: .
-      target: production
-      dockerfile: ./Dockerfile
-    command: npm run start:prod
+      - "8080:3000"
+      - "9229:9229"
+
+  postgres:
+    image: postgres:10.4
     ports:
-      - 3000:3000
-      - 9229:9229
-    networks:
-      - nesjs-network
+      - "35000:5432"
+    environment:
+      POSTGRES_USER: xyz
+      POSTGRES_PASSWORD: xyz
+      POSTGRES_DB: db
+
     volumes:
```

Comment:
Since we need to have persistent data, we need to volume map data in postgres container to our container. Research about this and have a clear understanding.